### ISSUE TITLE
Results dont follow the expected format

### ISSUE THREAD

**Issue by queicherius** [peermusic/scrape-engine#3]
Current results:

``` js
{
    "title": "Immigrant Song",
    "result": [
        {
            "href": "/music/Led+Zeppelin/_/Celebration+Day",
            "content": "Celebration Day"
        },
        // ...
    ]
}
```

Expected results:

``` js
{
    "title": "Immigrant Song",
    "result": [
        {
            // Title, album and artist are required
            // Track, year and genre are optional
            "title": "Celebration Day",
            "album": "Led Zeppelin III",
            "artist": "Led Zeppelin",
            "track": 3,
            "year": 1970,
            "genre": "classic rock"
        },
        // ...
    ]
}
```

**Comment by ryomiyaichi** [comment]
Fixed the format: a99829c90fef44a09f40780a262a30c60a48c2bd

**Comment by queicherius** [comment]
:+1:
